packages/plugins/src/plugins/curve/getPositionsFetcher.ts

```diff
@@ -87,13 +88,21 @@ export function getPositionsFetcher(crvNetworkId: CrvNetworkId): Fetcher {
         const shares = balance.div(supply);
         const assets: PortfolioAssetToken[] = coins
           .map((coin) => {
+            /* 
+                On Mainnet some pools use the EIP-7528 Ether address instead of zeroAddress to represent Ether.
+                This could cause issues on the octav server, since we use the zeroAddress to represent Ether.
+            */
+            const resolvedAddress =
+              getAddress(coin.address) === getAddress(eip7528EthAddress)
```

Comment:
```suggestion
              getAddress(coin.address) ===eip7528EthAddress
```

Given the new constant's name, let's properly type it to `Address` and avoid calling `getAddress()` in a loop.